### ISSUE TITLE
When someone is typing in chat, show a "typing" status

### ISSUE THREAD

**Issue by bigbluebutton-issue-import** [bigbluebutton/bigbluebutton#1320]
Originally reported on Google Code with ID 565

```
When someone is entering text in chat, it would be nice to see an indication that they
are typing. It would make things feel a little more interactive.
```

Reported by `fishnscout` on 2010-07-06 22:27:05

**Comment by Jarvil** [comment]
Hello,

I just tested this with BBB but I don't see "typing" message anywhere, where is it supposed to show?

**Comment by ffdixon** [comment]
This feature was in the Flash client.  Opened a new issue to add it to the HTML5 client https://github.com/bigbluebutton/bigbluebutton/issues/6851.

**Comment by Jarvil** [comment]
Hello,

I'm using the flash client and I don't see it.

**Comment by ffdixon** [comment]
We implemented this in BigBlueButton 2.2-beta.  You can see it by going to https://test.bigbluebutton.org/ and login with two clients (both Flash).  Type on one and you'll see the message in the other.

![image](https://user-images.githubusercontent.com/290351/53323835-459e2300-38ad-11e9-9cc6-6d74d92a16ed.png)

We are in process of moving this to the HTML5 client.
